foundation-extensions/mobile-core/identity/identity-event-reference.md

```diff
@@ -266,14 +266,14 @@ The result for the `getUrlVariables` request:
 ```
 
 {% hint style="warning" %}
-The Analytics Custom VID is no longer included in the Identity event data payload. If needed, it may be retrieved from the [Analytics extension](https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/tree/987092ee0fd2699ae5898755cd72255802628107/using-mobile-extensions/adobe-analytics/analytics-api-reference/README.md#get-the-custom-visitor-identifier).
+The Analytics Custom VID is no longer included in the Identity event data payload. If needed, it may be retrieved from the [Analytics extension](../../../using-mobile-extensions/adobe-analytics/analytics-api-reference.md#get-the-custom-visitor-identifier).
 {% endhint %}
 
 ### Configuration content request
 
 The Identity extension dispatches a new Configuration Content Request event when a privacy change is detected in the response that is received from the ECID Service. This event is handled by the Core extension, and the new privacy status is broadcasted to other extensions.
 
-For more details about this event, see the [Mobile Core](https://app.gitbook.com/@aep-sdks/s/docs/~/drafts/-LzsATEkeUMSPZRDl_O0/using-mobile-extensions/mobile-core) section.
+For more details about this event, see the [Mobile Core Configuration event reference](../configuration/configuration-event-reference) section.
```

Comment:
can you check if this goes to the correct link?

Reply:
The link is correct and will point to this page, https://aep-sdks.gitbook.io/docs/foundation-extensions/mobile-core/configuration/configuration-event-reference. Unfortunately, there is no content, but it is the correct page to link to.